include/ginkgo/core/base/range.hpp

```diff
@@ -686,15 +686,15 @@ GKO_ENABLE_UNARY_RANGE_OPERATION(zero_operation, zero,
                                  accessor::detail::zero_operation);
 GKO_ENABLE_UNARY_RANGE_OPERATION(one_operaton, one,
                                  accessor::detail::one_operation);
-GKO_ENABLE_UNARY_RANGE_OPERATION(abs_operaton, abs,
+GKO_ENABLE_UNARY_RANGE_OPERATION(abs_operation, abs,
```

Comment:
You can do the following to prevent the interface break:
```c++
#define GKO_DEPRECATED_UNARY_RANGE_OPERATION(_operation_deprecated_name,     \
                                             _operation_name)                \
    namespace accessor {                                                     \
    template <typename Operand>                                              \
    struct [[deprecated(                                                     \
        "Please use " #_operation_name)]] _operation_deprecated_name         \
        : _operation_name<Operand>{};                                        \
    }                                                                        \
    static_assert(true,                                                      \
                  "This assert is used to counter the false positive extra " \
                  "semi-colon warnings")

...

GKO_ENABLE_UNARY_RANGE_OPERATION(abs_operation, abs,
                                 accessor::detail::abs_operation);
GKO_DEPRECATED_UNARY_RANGE_OPERATION(abs_operaton, abs_operation);
```